src/commands/hook/hook.js

```diff
@@ -3,8 +3,13 @@ const HOOK: Object = {
   PERMISSIONS: 0o775,
   FILENAME: 'prepare-commit-msg',
   CONTENTS:
-    '#!/bin/sh\n# gitmoji as a commit hook\n' +
-    'exec < /dev/tty\ngitmoji --hook $1 $2\n'
+    '#!/usr/bin/env sh\n# gitmoji as a commit hook\n' +
+    'if npx -v >/dev/null\n' +
```

Comment:
Wait! I think we need to redirect std error as well, otherwise if the command is not available it will show an error:

I believe we should check the status code of the command:

https://www.cyberciti.biz/faq/bash-get-exit-code-of-command/

```suggestion
    'if npx -v >&/dev/null\n' +
```

Reply:
@carloscuesta thanks for the trigger. Sorry I totally forgot about that PR..
Updated your suggestions.